packages/react-data-grid/src/__tests__/Cell.spec.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { PropsWithChildren } from 'react';
+import { shallow } from 'enzyme';
+
+import Cell, { Props } from '../Cell';
+import helpers from '../helpers/test/GridPropHelpers';
+import CellAction from '../CellAction';
+import { SimpleCellFormatter } from '../formatters';
+import { Column, CellMetaData } from '../common/types';
+
+const testCellMetaData: CellMetaData = {
+  rowKey: 'row',
+  onCellClick() {},
+  onCellContextMenu() {},
+  onCellDoubleClick() {},
+  onDragEnter() {},
+  onCellExpand() {},
+  onRowExpandToggle() {}
+};
+
+const expandableOptions = {
+  canExpand: false,
+  field: 'field',
+  expanded: false,
+  children: [],
+  treeDepth: 0,
+  subRowDetails: {
+    canExpand: false,
+    field: 'field',
+    expanded: false,
+    children: [],
+    treeDepth: 0,
+    siblingIndex: 0,
+    numberSiblings: 0
+  }
+};
+
+const defaultColumn: Column = { key: 'col1', name: 'col1', width: 100, left: 0 };
+
+const testProps: Props = {
+  rowIdx: 0,
+  idx: 1,
+  column: defaultColumn,
+  value: 'Wicklow',
+  cellMetaData: testCellMetaData,
+  rowData: { name: 'Johnny Test', location: 'Wicklow', likesTesting: 'Absolutely' },
+  height: 40,
+  isScrolling: false,
+  scrollLeft: 0
+};
+
+const renderComponent = (extraProps?: PropsWithChildren<Partial<Props>>) => {
+  return shallow<Cell>(<Cell {...testProps} {...extraProps} />);
+};
+
+describe('Cell Tests', () => {
+  it('should render a SimpleCellFormatter with value', () => {
+    const wrapper = renderComponent();
+    const formatter = wrapper.find(SimpleCellFormatter);
+    expect(formatter.props().value).toEqual('Wicklow');
+  });
+
+  it('should render a custom formatter when specified on column', () => {
+    const CustomFormatter = () => <div>Custom</div>;
+
+    const column = {
+      ...defaultColumn,
+      formatter: CustomFormatter
+    };
+
+    const wrapper = renderComponent({ column });
+    const formatterInstance = wrapper.find(CustomFormatter);
+    expect(formatterInstance.prop('value')).toEqual('Wicklow');
+  });
+
+  it('should render children when those are passed', () => {
+    const wrapper = renderComponent({ children: <div>Child</div> });
+    expect(wrapper.children().text()).toBe('Child');
+  });
+
+  describe('Rendering Cell component', () => {
+    function shallowRenderComponent(props: Props) {
```

Comment:
consider reusing this https://github.com/adazzle/react-data-grid/pull/1602/files#diff-193c6bb6a70f1703324d0ea5dff20df5R51 ?

Reply:
The other function is using `testProps`, so I think it's safe to keep it that way to avoid any possible future conflicts.